README.md

```diff
@@ -1,8 +1,12 @@
 # Autoretrieve
 
-Autoretrieve is a standalone gateway server / Bitswap provider that bridges Filecoin Graphsync content over to Bitswap clients. 
+Autoretrieve is a standalone Graphsync-to-Bitswap proxy server that allows IPFS clients to retrieve data which may be available on the Filecoin network but not on IPFS. 
 
-When a Bitswap client requests data from Autoretrieve, Autoretrieve asks either Estuary or an STI indexer which Filecoin service providers are hosting it, starts a retrieval deal with a selected SP, and streams the incoming blocks to the Bitswap client.
+## What problem does Autoretrieve solve?
+
+Protocol Labs develops two related but incompatible decentralized data transfer protocols - Bitswap and GraphSync, which back the IPFS and Filecoin networks, respectively. These networks are fundamentally incompatible - a client of one network cannot retrieve data from a provider of the other. [TODO: link more info about differences.] This raises the following issue: what if there is data that exists on Filecoin, but not on IPFS?
+
+Autoretrieve is a "translational proxy" that allows data to be transferred from Filecoin to IPFS in an automated fashion. The existing alternatives for Autoretrieve's Filecoin-to-IPFS flow are the Boost IPFS node that providers may optionally enable, and manual transfer.
```

Comment:
> The existing alternatives for Autoretrieve's Filecoin-to-IPFS flow are the Boost IPFS node that providers may optionally enable

What's the difference between using Autoretrieve and the Boost IPFS node?

Reply:
@anjor i added a link to the boost IPFS github issue (since it does not seem to be released yet)